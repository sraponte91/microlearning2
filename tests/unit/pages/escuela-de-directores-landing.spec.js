import { config, createLocalVue, mount } from '@vue/test-utils'
import VueMeta from 'vue-meta'
import EscuelaDeDirectoresLanding from '@/pages/escuela-de-directores-landing.vue'
// create vue with vue-meta
const localVue = createLocalVue()
localVue.use(VueMeta, { keyName: 'head' })

const mock = {
    localVue,
    mocks: {}
}

const wrapper = mount(EscuelaDeDirectoresLanding, mock)

describe('escuela-de-directores-landing.vue', () => {
    test('Is a Vue instance', () => {
        expect(wrapper.vm).toBeTruthy()
    })
    test('Renders correctly', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
    test('has correct <head> content', () => {
        expect(wrapper.vm.$metaInfo.title).toBe(config.mocks['$constants'].escuelaDeDirectoresLanding.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'title'
        ).content).toBe(config.mocks['$constants'].escuelaDeDirectoresLanding.meta.metaTitle)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'description'
        ).content).toBe(config.mocks['$constants'].escuelaDeDirectoresLanding.meta.metaDescription)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:title'
        ).content).toBe(config.mocks['$constants'].escuelaDeDirectoresLanding.meta.ogTitle)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:description'
        ).content).toBe(config.mocks['$constants'].escuelaDeDirectoresLanding.meta.ogDescription)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:image'
        ).content).toBe(config.mocks['$constants'].baseURL + config.mocks['$constants'].escuelaDeDirectoresLanding.meta.ogImage)
    })
})
