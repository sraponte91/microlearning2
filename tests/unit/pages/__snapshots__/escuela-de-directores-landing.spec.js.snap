// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`escuela-de-directores-landing.vue Renders correctly 1`] = `
<main>
  <section
    class="director-section-1"
  >
    <div
      class="container margin-0-auto"
    >
      <div
        class="row"
      >
        <div
          class="col-md-12 text-center leader-title"
        >
          <h1
            class="ff-sans-b"
          >
            Escuela de directores
          </h1>
        </div>
         
        <div
          class="col-md-12"
        >
          <div
            class="sec1-boxes"
          >
            <img
              alt=""
              class="img-fluid"
              src="/images/director-about.jpg"
            />
             
            <div
              class="sec1-description"
            >
              <p>
                
                Escuela de Directores Aflora, es un proyecto de la Fundación
                Bolívar Davivienda, que reúne a los líderes de organizaciones
                sociales que están comprometidos en trabajar con pasión por su
                causa. El proceso formativo se realiza con EDIME,la Unidad de
                Gerencia Media del INALDE, Escuela de Negocios de la
                Universidad de la Sábana, cuyo objetivo es formar de manera
                integral a quienes ocupan cargos directivos dentro de las
                organizaciones
              
              </p>
               
              <p>
                
                Es este espacio damos herramientas para que el talento y la
                disciplina ayuden a lograr el éxito de las causas que lideran,
                tomando como fundamento el Método del Caso, donde exploramos 8
                modelos nuevos a partir de experiencias organizacionales de
                participantes en el programa.
              
              </p>
            </div>
          </div>
           
          <button
            class="btn btn-primary"
            type="button"
          >
            Inscríbase aquí
          </button>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="director-section-2 bg-grey"
  >
    <div
      class="container margin-0-auto"
    >
      <div
        class="row"
      >
        <div
          class="col-md-8 offset-md-2"
        >
          <h2
            class="ff-sans-b"
          >
            Nuestros expertos
          </h2>
           
          <img
            alt=""
            class="img-fluid"
            src="/images/director-expert.jpg"
          />
           
          <button
            class="btn btn-primary"
            type="button"
          >
            Conozca más
          </button>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="director-section-3"
  >
    <div
      class="container margin-0-auto"
    >
      <div
        class="row"
      >
        <div
          class="col-md-8 offset-md-2"
        >
          <h2
            class="ff-sans-b"
          >
            Método del caso
          </h2>
        </div>
         
        <div
          class="col-md-12"
        >
          <div>
            <div
              class="VueCarousel"
            >
              <div
                class="VueCarousel-wrapper"
              >
                <div
                  class="VueCarousel-inner"
                  style="transform: translate(0px, 0); transition: 0.5s ease transform; flex-basis: 0px; visibility: hidden; height: auto;"
                >
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative box bg-white"
                    >
                      <div
                        class="main-slider-at-top actualidad-pg"
                      >
                        <div
                          class="img-prt position-relative"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/escuela-directores-landing/slide_01.jpg"
                          />
                           
                          <div
                            class="container position-absolute set-to-bottom"
                          >
                            <div
                              class="banner-txt text-white"
                            >
                              <div
                                class="orange-bg"
                              >
                                <h3
                                  class="ff-sans-b"
                                >
                                  Unidad 1
                                </h3>
                              </div>
                               
                              <div
                                class="black-bg"
                              >
                                <p
                                  class="ff-nunito"
                                >
                                  Método del caso
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                     
                    <div
                      class="sec3-info"
                    >
                      <p>
                        Es un innovador proceso de aprendizaje y perfeccionamiento de capacidades directivas que se trabaja en tres niveles.
                      </p>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative box bg-white"
                    >
                      <div
                        class="main-slider-at-top actualidad-pg"
                      >
                        <div
                          class="img-prt position-relative"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/escuela-directores-landing/slide_02.jpg"
                          />
                           
                          <div
                            class="container position-absolute set-to-bottom"
                          >
                            <div
                              class="banner-txt text-white"
                            >
                              <div
                                class="orange-bg"
                              >
                                <h3
                                  class="ff-sans-b"
                                >
                                  Unidad 2
                                </h3>
                              </div>
                               
                              <div
                                class="black-bg"
                              >
                                <p
                                  class="ff-nunito"
                                >
                                  Estudio Individual
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                     
                    <div
                      class="sec3-info"
                    >
                      <p>
                        Los participantes resuelven los retos que plantean cada uno de los casos, diagnosticando e identificando el problema, proponiendo individualmente alternativas de solución, estableciendo los criterios de evaluación y tomando decisiones que permitan diseñar un plan de acción. Se estima una dedicación promedio de 120 minutos por caso.
                      </p>
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    class="VueCarousel-slide"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative box bg-white"
                    >
                      <div
                        class="main-slider-at-top actualidad-pg"
                      >
                        <div
                          class="img-prt position-relative"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/escuela-directores-landing/slide_03.jpg"
                          />
                           
                          <div
                            class="container position-absolute set-to-bottom"
                          >
                            <div
                              class="banner-txt text-white"
                            >
                              <div
                                class="orange-bg"
                              >
                                <h3
                                  class="ff-sans-b"
                                >
                                  Unidad 3
                                </h3>
                              </div>
                               
                              <div
                                class="black-bg"
                              >
                                <p
                                  class="ff-nunito"
                                >
                                  Trabajo en equipo
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                     
                    <div
                      class="sec3-info"
                    >
                      <p>
                        Los participantes se reúnen en grupos de 8 a 10 líderes, para analizar en equipo los casos como comité de dirección. En él expondrán sus puntos de vista y las conclusiones a las que llegaron a través del estudio individual, contrastando así sus alternativas y opiniones. Cada grupo contará con un monitor y 20 minutos para evaluar cada caso.
                      </p>
                    </div>
                  </div>
                  <div
                    class="VueCarousel-slide VueCarousel-slide-active VueCarousel-slide-center"
                    role="tabpanel"
                    tabindex="-1"
                  >
                    <div
                      class="position-relative box bg-white"
                    >
                      <div
                        class="main-slider-at-top actualidad-pg"
                      >
                        <div
                          class="img-prt position-relative"
                        >
                          <img
                            alt=""
                            class="img-fluid"
                            src="/images/escuela-directores-landing/slide_04.jpg"
                          />
                           
                          <div
                            class="container position-absolute set-to-bottom"
                          >
                            <div
                              class="banner-txt text-white"
                            >
                              <div
                                class="orange-bg"
                              >
                                <h3
                                  class="ff-sans-b"
                                >
                                  Unidad 4
                                </h3>
                              </div>
                               
                              <div
                                class="black-bg"
                              >
                                <p
                                  class="ff-nunito"
                                >
                                  Sesión general
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                     
                    <div
                      class="sec3-info"
                    >
                      <p>
                        En la modalidad de asamblea, esta reunión con todos los participantes será dirigida por un experto en el método del caso, con amplio conocimiento en dirección empresarial estratégica, que conducirá la argumentación ordenada y metodología rigurosa para enriquecer las conclusiones. En la sesión, cada uno ratificará o modificará su punto de vista acerca de la situación que plantea el caso y analizará sus decisiones. Tendrá una duración de una hora y veinte minutos.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
               
              <!---->
               
              <div
                class="VueCarousel-pagination"
                data-v-438fd353=""
                style=""
              >
                <div
                  class="VueCarousel-dot-container"
                  data-v-438fd353=""
                  role="tablist"
                  style="margin-top: 20px;"
                >
                  <button
                    aria-hidden="false"
                    aria-label="Item 0"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 0"
                    value="Item 0"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 1"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 1"
                    value="Item 1"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 2"
                    aria-selected="false"
                    class="VueCarousel-dot"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(239, 239, 239);"
                    title="Item 2"
                    value="Item 2"
                  />
                  <button
                    aria-hidden="false"
                    aria-label="Item 3"
                    aria-selected="true"
                    class="VueCarousel-dot VueCarousel-dot--active"
                    data-v-438fd353=""
                    role="tab"
                    style="margin-top: 20px; padding: 10px; width: 10px; height: 10px; background-color: rgb(0, 0, 0);"
                    title="Item 3"
                    value="Item 3"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="director-section-4 bg-grey"
  >
    <div
      class="container margin-0-auto"
    >
      <div
        class="row"
      >
        <div
          class="col-md-6"
        >
          <h2
            class="ff-sans-b"
          >
            Pénsum
          </h2>
           
          <p>
            
            El Seminario de Escuela de Directores - Fundación Bolívar
            Davivienda - EDIME, comprende las siguientes áreas:
          
          </p>
           
          <ul>
            <li>
              Análisis de situaciones de negocios
            </li>
             
            <li>
              Dirección de personas en las organizaciones
            </li>
             
            <li>
              Consecución de recursos
            </li>
             
            <li>
              Cultura organizacional
            </li>
             
            <li>
              Gestión de procesos
            </li>
             
            <li>
              Estrategia
            </li>
          </ul>
        </div>
         
        <div
          class="col-md-6"
        >
          <img
            alt=""
            class="img-fluid"
            src="/images/director-pensum.jpg"
          />
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="director-section-5"
  >
    <div
      class="container margin-0-auto"
    >
      <div
        class="row"
      >
        <div
          class="col-md-8 offset-md-2 text-cente"
        >
          <p
            class="ff-nunito caption"
          >
            Estructura del
          </p>
           
          <h2
            class="ff-sans-b"
          >
            Seminario
          </h2>
           
          <div
            class="row"
          >
            <div
              class="col-md-12"
            >
              <h3
                class="ff-sans-b"
              >
                Fechas
              </h3>
               
              <p>
                
                La Escuela de directores la haremos el 18,19, 20,23,24,25 y 26 de Agosto en un horario de 8:00 am a 12:00.
              
              </p>
            </div>
             
            <div
              class="col-md-12"
            >
              <br />
            </div>
             
            <div
              class="col-md-12"
            >
              <h3
                class="ff-sans-b"
              >
                Actividades
              </h3>
               
              <p>
                
                  El primer día se llevarán a cabo la inauguración, la introducción y la presentación a cargo de los directivos de la Fundación Bolívar Davivienda, y 1 sesión de trabajo.
                
              </p>
               
              <p>
                
                  Los seis  días siguientes habrá  entre 2 y 3 sesiones, para un total de 14. El último día celebraremos el cierre del seminario y se entregarán los certificados de asistencia.
                
              </p>
            </div>
             
            <div
              class="col-md-12"
            >
              <br />
            </div>
             
            <div
              class="col-md-12"
            >
              <h3
                class="ff-sans-b"
              >
                Tiempo dedicado
              </h3>
               
              <p>
                
                  Demandará una dedicación de 30 horas presenciales y aproximadamente 20 horas de estudio individual, para preparar los casos antes de cada sesión y lograr una mejor interacción y participación por parte de los asistentes.
                
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="director-section-6 bg-grey"
  >
    <div
      class="container margin-0-auto"
    >
      <p
        class="ff-nunito caption"
      >
        Perfil de los
      </p>
       
      <h2
        class="ff-sans-b"
      >
        participantes
      </h2>
       
      <div
        class="row"
      >
        <div
          class="col-md-4"
        >
          <div
            class="sec6-inner-box"
          >
            <h4
              class="ff-sans-b"
            >
              1. Experiencia Mínima
            </h4>
             
            <img
              alt=""
              class="img-fluid"
              src="/images/director-participate-1.jpg"
            />
             
            <p>
              
              Experiencia de mínimo 3 años en nivel directivo o gerencial en
              posiciones de responsabilidad, preferiblemente dentro de una
              fundación como director de área, director de la organización
              social, miembro del consejo, junta directiva o representante
              legal.
            
            </p>
          </div>
        </div>
         
        <div
          class="col-md-4"
        >
          <div
            class="sec6-inner-box"
          >
            <h4
              class="ff-sans-b"
            >
              2. Disponibilidad de tiempo
            </h4>
             
            <img
              alt=""
              class="img-fluid"
              src="/images/director-participate-2.jpg"
            />
             
            <p>
              
              Disponibilidad para asistir virtualmente mediante la plataforma
              de Zoom los 7 días del evento de 8:00 a. m. a 12:00p. m.
            
            </p>
          </div>
        </div>
         
        <div
          class="col-md-4"
        >
          <div
            class="sec6-inner-box"
          >
            <h4
              class="ff-sans-b"
            >
              3. Formulario Completo
            </h4>
             
            <img
              alt=""
              class="img-fluid"
              src="/images/director-participate-3.jpg"
            />
             
            <p>
              
              Diligenciar el formato de inscripción en donde también deberán
              responder preguntas sobre su hoja de vida para iniciar con el
              proceso de selección. Deben completar todos los campos; los
              formularios incompletos no serán tenidos en cuenta.
            
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="director-section-6"
  >
    <div
      class="container margin-0-auto"
    >
      <p
        class="ff-nunito caption"
      >
        Costos
      </p>
       
      <h2
        class="ff-sans-b"
      >
        2021
      </h2>
       
      <div
        class="row justify-content-center"
      >
        <div
          class="col-md-5"
        >
          <div
            class="sec6-inner-box"
          >
            <h4
              class="ff-sans-b text-center"
            >
              Por persona
            </h4>
             
            <img
              alt=""
              class="img-fluid"
              src="/images/director-costos-1.jpg"
            />
             
            <p>
              
              El costo por persona será de 
              <strong>
                $1.800.000
              </strong>
               y está
              dirigido a colaboradores que hagan parte de las organizaciones
              de Aflora. 
              <strong>
                Incluye
              </strong>
               participación los 7 días,
              materiales, y certificado.
            
            </p>
             
            <p>
              
              El pago se debe realizar a Edime 
              <strong>
                de contado
              </strong>
               y
              se entregará sólo una factura.
            
            </p>
          </div>
        </div>
         
        <div
          class="col-md-5"
        >
          <div
            class="sec6-inner-box"
          >
            <h4
              class="ff-sans-b text-center"
            >
              3. Formulario Completo
            </h4>
             
            <img
              alt=""
              class="img-fluid"
              src="/images/director-costos-2.jpg"
            />
             
            <p>
              
              El pago se debe realizar directamente a una cuenta bancaria de
              EDIME. La Fundación Bolívar Davivienda informará el número de
              cuenta bancaria para que el participante pueda hacer la
              consignación correspondiente, y posteriormente enviar el
              comprobante de pago a
              
              <a
                href="mailto:escueladirectores@fundacionbd.org"
              >
                escueladirectores@fundacionbd.org
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</main>
`;
